test/JDBC/expected/sys-type_info_internal-dep-vu-prepare.out

```diff
@@ -0,0 +1,31 @@
+-- psql 
```

Comment:
There is one function which would return pg_type to tsql_type list,
```
CREATE OR REPLACE FUNCTION sys.babelfish_typecode_list()
RETURNS table (
  oid int,
  pg_namespace text,
  pg_typname text,
  tsql_typname text,
  type_family_priority smallint,
  priority smallint,
  sql_variant_hdr_size smallint
) AS 'babelfishpg_common', 'typecode_list' LANGUAGE C;
```
Let's add test case which would verify count of entries in C table vs count of entries in the internal view table_types_internal. This will ensure that C table and internal view remain in sync. May be we can think of some variations around this.

Reply:
Somehow I missed this function. Thank you for pointing it out.

In fact, I changed the implementation of the sys.types so that it uses the sys.babelfish_typecode_list function instead of the sys.type_info_internal view. The performance is the same.

I removed the sys.type_info_internal view so we don't have to worry about keeping it in sync with the entries in C table.